static/app/views/automations/edit.tsx

```diff
@@ -157,17 +160,21 @@ function AutomationEditForm({automation}: {automation: Automation}) {
       <AutomationDocumentTitle />
       <Layout.Page>
         <StyledLayoutHeader>
-          <Layout.HeaderContent>
-            <AutomationBreadcrumbs automationId={params.automationId} />
-            <Layout.Title>
-              <EditableAutomationName />
-            </Layout.Title>
-          </Layout.HeaderContent>
-          <Flex>
-            <EditAutomationActions automation={automation} />
-          </Flex>
+          <HeaderInner maxWidth={maxWidth}>
+            <Layout.HeaderContent>
+              <AutomationBreadcrumbs automationId={params.automationId} />
+              <Layout.Title>
+                <EditableAutomationName />
+              </Layout.Title>
+            </Layout.HeaderContent>
+            <div>
+              <Flex>
+                <EditAutomationActions automation={automation} />
+              </Flex>
+            </div>
+          </HeaderInner>
         </StyledLayoutHeader>
-        <Layout.Body>
+        <StyledBody maxWidth={maxWidth}>
```

Comment:
Will be able to add a prop Layout.Main once this goes in:

https://github.com/getsentry/sentry/pull/101607